consul-lambda-registrator/consul-lambda-extension/main.go

```diff
@@ -0,0 +1,125 @@
+package main
```

Comment:
What do you think about moving this outside of `consul-lambda-registrator`?

Reply:
Moving stuff around can definitely happen in a follow up PR

Reply:
Yes, absolutely! I will do that in a follow up PR. My proposed layout is:
- terraform-aws-consul-lambda/
  - consul-lambda/
    - go.mod
    - go.sum
    - client/
    - consul-lambda-registrator/
    - consul-lambda-extension/
    - proxy/
    - structs/

Thoughts?